docs/how-to-open-a-pull-request.md

```diff
@@ -50,7 +50,7 @@ Keep it short (less than 30 characters) and simple; you can add more information
 Some examples of good PR titles would be:
 
 - `fix(a11y): improved search bar contrast`
-- `feat: add more tests to HTML and CSS challenges`
```

Comment:
Revert this; scopes are indeed optional.

Reply:
I have seen good prs name, but was blocked by scope